facility-all-programs.ipynb

```diff
@@ -424,16 +572,16 @@
     "        have_water_enf = True\n",
     "        # print( water_enf_data )\n",
     "    except pd.errors.EmptyDataError:\n",
-    "        print( \"No data for this facility in NPDES_FORMAL_ENFORCEMENT_ACTIONS.\")\n",
+    "        print( \"This is a CWA-regulated facility, but there's no data for it in NPDES_FORMAL_ENFORCEMENT_ACTIONS.\")\n",
     "    "
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Show the information from one of the data sets for this facility\n",
-    "Running this cell will create a dropdown list of the program data sets available for this faciliity. Pick one and then run the second cell to view its contents. To see additional data, return to the dropdown to choose a new value and repeat."
+    "### Show the information from one of the data sets for this facility\n",
+    "Running this cell will create a dropdown list of the program data sets available specifically for this faciliity. Pick one and then run the second cell to view its contents. To see additional data, return to the dropdown to choose a new value and repeat."
```

Comment:
The "to see additional data" might make more sense after the cell's output, like you did above

Reply:
done!